elastic/beats
# [Heartbeat] Add status to monitor run log struct

## What does this PR do?



Closes #31869. Add status to monitor log list.

## Checklist




## How to test this PR locally

Run monitors heartbeat locally and check that status is added to each run log:
```
{"file.name":"logger/logger.go","file.line":125},"message":"Monitor finished","service.name":"heartbeat",
"event":{"action":"monitor.run"},"monitor":{"id":"my-monitor-797ee71f1ee5821c","type":"http","status":"up",
"duration":{"ms":4012}}
```
